src/libmongoc/src/mongoc/mongoc-host-list.c

```diff
@@ -358,11 +372,14 @@ _mongoc_host_list_from_hostport_with_err (mongoc_host_list_t *link_,
       link_->family = AF_UNSPEC;
 
       mongoc_lowercase (link_->host, link_->host);
-      bson_snprintf (link_->host_and_port,
-                     sizeof link_->host_and_port,
-                     "%s:%hu",
-                     link_->host,
-                     link_->port);
+      int req = bson_snprintf (link_->host_and_port,
+                               sizeof link_->host_and_port,
+                               "%s:%hu",
```

Comment:
```suggestion
                               "%s:%" PRIu16,
```
For consistency with `uint16_t` (from `<inttypes.h>`).